subworkflows/local/prepare_alphafold2_dbs.nf

```diff
@@ -40,7 +40,7 @@ workflow PREPARE_ALPHAFOLD2_DBS {
         ch_mmcif_files = file( params.pdb_mmcif_path, type: 'dir' )
         ch_mmcif_obsolete = file( params.pdb_mmcif_path, type: 'file' )
         ch_mmcif = ch_mmcif_files + ch_mmcif_obsolete
-        ch_uniclust30 = file( params.uniclust30_path, type: 'dir' )
+        ch_uniref30   = file( params.uniref30_alphafold2_path, type: 'any' )
```

Comment:
If my first comment is OK (the one that has been to be corroborated by @athbaltzis then this should be):
```suggestion
        ch_uniref30   = file( params.uniref30_alphafold2_path, type: 'dir' )
```

Reply:
I am not 100% sure of the different behaviour between the two, I can tell you that there are a bunch of files in that directory (no sub-directories), so I guess that dir might be more accurate

Reply:
By the book, this should have `type: 'file'`. But `type: 'any'` would work as well.

Reply:
let me know what you guys prefer :)

Reply:
If `any` work we could leave it as it is 😄